components/console/logger.rst

```diff
@@ -85,6 +85,8 @@ through the second parameter of the :class:`Symfony\\Component\\Console\\Console
 constructor::
 
     // ...
+    use Psr\Log\LogLevel;
+    // ...
```

Comment:
@xabbuh why I added it was for everyone knows there is something up and down. Still if you feel I will move .